### PR TITLE
versions.yaml updated for release 0.30 - final release

### PR DESCRIPTION
**Pull request by vinayada1**
# Description

versions.yaml updated for release 0.30 - final release

## Type of change

- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Fixes: #issue_number

### CODE REVIEW

**Review by kachawla — DISMISSED** [review]

**Review by kachawla — COMMENTED** [review]
This should include release notes as well.. or have we changed the process around that?

**Comment by AaronCrawfis** [comment]
Release note highlights:

```markdown
### Terraform Recipe enhancements

We’ve added a bunch of new supported features and scenarios for Terraform Recipes, including sub-module support, Git module source support, and version ranges. We’re actively working on additional features such as private modules and any provider. Stay tuned for updates! Learn more in [our docs](https://docs.radapp.io/guides/recipes/overview/).

### Radius roadmap

The Radius roadmap is now published at https://aka.ms/radius-roadmap, where you can keep track of all the major features and improvements we are tracking. Check it out and make sure to upvote and leave feedback on your favorites!

### Introducing the Radius dashboard

We’re excited to share a first look at a new experimental feature of Radius: the Radius dashboard! Our new dashboard allows you to interact with Radius environments, applications, and recipes from an easy-to-use graphical interface. Visit the [dashboard repo](https://github.com/radius-project/dashboard) to try it out today.
```

File: docs/release-notes/v0.30.0.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+None
+
+## New contributors
+
+Welcome to our new contributors who have merged their first PR in this release!
+
+* @IvanJosipovic made their first contribution in https://github.com/radius-project/radius/pull/6996
+* @gpltaylor made their first contribution in https://github.com/radius-project/radius/pull/6974
+* @akshayrohilla made their first contribution in https://github.com/radius-project/radius/pull/7083
+* @jhandel made their first contribution in https://github.com/radius-project/radius/pull/7095
```
Comment (AaronCrawfis) [inline docs/release-notes/v0.30.0.md:26]: This got merged after the RC was cut. Is this code in 0.30, or do we have a bug in our release note generation script?

File: docs/release-notes/v0.30.0.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+<!-- TALK TO THE PM TEAM ABOUT WHAT HIGHLIGHTS TO ADD HERE -->
```
Comment (AaronCrawfis) [inline docs/release-notes/v0.30.0.md:13]: Added these as a comment to the PR

**Comment by vinayada1** [comment]
Release notes are not automatically generated due to a recent change to the workflow file. Created a bug: https://github.com/radius-project/radius/issues/7106

For this release, removed the change to the release.yaml workflow file which causes this step to be skipped to generate the release notes

File: docs/release-notes/v0.30.0.md (hunk 1 above)
Comment (vinayada1) [reply docs/release-notes/v0.30.0.md:26]: Looks like a bug in the release notes generation. @IvanJosipovic made their first contribution in https://github.com/radius-project/radius/pull/6996 is not present in this branch (release-0.30)

Comment (vinayada1) [reply docs/release-notes/v0.30.0.md:26]: Looks like a release notes generation issue. This is not present in this branch:-
* @jhandel made their first contribution in https://github.com/radius-project/radius/pull/7095

Comment (vinayada1) [reply docs/release-notes/v0.30.0.md]: Created: https://github.com/radius-project/radius/issues/7107 to address this

**Review by kachawla — APPROVED** [review]

**Pull request merged**